engine/src/main/java/io/camunda/zeebe/engine/state/migration/DbMigratorImpl.java

```diff
@@ -143,8 +144,13 @@ private void runMigration(final MigrationTask migrationTask, final int index, fi
     migrationTask.runMigration(processingState);
     final var duration = System.currentTimeMillis() - startTime;
 
+    markMigrationFinished(migrationTask.getIdentifier());
     LOGGER.debug(migrationTask.getIdentifier() + " migration completed in " + duration + " ms.");
     LOGGER.info(
         "Finished " + migrationTask.getIdentifier() + " migration (" + index + "/" + total + ")");
   }
+
+  private void markMigrationFinished(final String identifier) {
+    processingState.getMigrationState().markMigrationFinished(identifier);
+  }
```

Comment:
🔧 Since this is just a single-line method, let's inline it. That way, it's clear where it is marked as finished, namely in the migration state. That part was lost from the simplified method name.